src/L2ArbitrumGovernorV2.sol

```diff
@@ -35,12 +35,12 @@ contract L2ArbitrumGovernorV2 is
   /// @notice address for which votes will not be counted toward quorum
   /// @dev    A portion of the Arbitrum tokens will be held by entities (eg the treasury) that
   ///         are not eligible to vote. However, even if their voting/delegation is restricted their
-  ///         tokens will still count towards the total supply, and will therefore affect the quorom.
+  ///         tokens will still count towards the total supply, and will therefore affect the quorum.
   ///         Restricted addresses should be forced to delegate their votes to this special exclude
-  ///         addresses which is not counted when calculating quorum
-  ///         Example address that should be excluded: DAO treasury, foundation, unclaimed tokens,
+  ///         address which is not counted when calculating quorum
+  ///         Example addresses that should be excluded: DAO treasury, foundation, unclaimed tokens,
   ///         burned tokens and swept (see TokenDistributor) tokens.
-  ///         Note that Excluded Address is a readable name with no code of PK associated with it, and thus can't vote.
+  ///         Note that Excluded Address is a readable name with no code or PK associated with it, and thus can't vote.
```

Comment:
I don't really understand this last line. What is `Excluded Address`? I may just be missing context

Reply:
It's an address where the treasury or other large voters can delegate to, and its voting power is excluded when calculating quorum. Yea, it's arbitrum specific.